canonical-web-and-design/ubuntu.com
# Add missing captcha script on server/thank-you form

## Done

- When #8862 was implemented, it missed this form, which made it un-submittable.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure the "Get Ubuntu Server pro tips" form on /download/server/thank-you can be submitted and returns a PDF.
